### ISSUE TITLE
connection refused 127.0.0.1:6379

### ISSUE THREAD

**Issue by linusyoung** [shadowsocks/shadowsocks-manager#580]
Please answer these questions before submitting your issue. Thanks!

(Please mention that if the issue you filed is solved, you may wish to close it by yourself. Thanks again.)

(PS, you can remove 3 lines above, including this one, before post your issue.)

### What version of shadowsocks-manager are you using?
0.32.4

### What operating system are you using?
Debian GNU/Linux 9 (stretch)

### What version of Node.js are you using?
v8.15.0

### What did you do?
ssmgr -c ~/.ssmgr/webgui.yml

### What did you see instead?
[ioredis] Unhandled error event: Error: connect ECONNREFUSED 127.0.0.1:6379
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1191:14)

### What is your config in detail (with all sensitive info masked)?
```yml
webgui:
             use: true
             skin: default
             host: 127.0.0.1
             port: 8080
             site: xxxxxxxx
````

**Comment by linusyoung** [comment]
any suggestion how to fix this?

**Comment by linusyoung** [comment]
didn't know redis is required from ^0.30

**Comment by myfingerhurt** [comment]
Found this behavior on Version 0.33.27

You may need install redis-server manually in official docker.
`apt install redis-server`

Then run redis-server before starting webui.
`redis-server --daemonize yes`

 redis-server is needed for master node.
`ssmgr -c /root/.ssmgr/webui.yml`

But not needed for slave node.
`ssmgr -c /root/.ssmgr/ss.yml`
